mmv1/products/logging/LogView.yaml

```diff
@@ -27,17 +27,31 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 # Skipping sweeper since this is a child resource.
 skip_sweeper: true
 description: 'Describes a view over log entries in a bucket.'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  skip_import_test: true
+  parent_resource_type: 'google_logging_log_view'
+  parent_resource_attribute: 'name'
+  import_format: ['{{%parent}}/locations/{{location}}/buckets/{{bucket}}/views/{{name}}', '{{name}}']
+  allowed_iam_role: 'roles/logging.admin'
+  method_name_separator: ':'
+  fetch_iam_policy_verb: :POST
+  set_iam_policy_verb: :POST
```

Comment:
I believe the default for `set_iam_policy_verb` is POST, so I think you can remove this line.

Reply:
Done